flink-connector-oceanbase-base/src/test/java/com/oceanbase/connector/flink/OceanBaseProxyContainer.java

```diff
@@ -74,7 +74,7 @@ public int getSqlPort() {
         return getMappedPort(SQL_PORT);
     }
 
-    public int getRpcPort() {
```

Comment:
It's better to change the data type at caller side.

Reply:
yes, I've modified it